frontend/public/components/utils/volume-icon.jsx

```diff
@@ -1,17 +1,26 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
 import {VolumeSource} from '../../module/k8s/pods';
+import { ResourceLink } from './resource-link';
 
-export const VolumeIcon = ({kind}) => {
+export const VolumeLink = ({kind, name, namespace}) => {
```

Comment:
Maybe call this `VolumeType` to match the table heading.

We should also rename this file if we're renaming the component.